app/controllers/api/v2/notifications_controller.rb

```diff
@@ -0,0 +1,40 @@
+module API
+  module V2
+    class NotificationsController < API::V2::ApplicationController
+      deserializable_resource :user_notification,
+                              class: API::V2::DeserializableNotification
+
+      def index
+        authorize @notifications = UserNotification.where(user_id: params[:user_id])
```

Comment:
given that there is `@current_user`

is there a real need to pass in the `user_id` as that kinda suggusts that they can be managed by anyone else, with out any bounded checks

Reply:
Yep I've changed